update.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+css='<link rel="stylesheet" href="index.css">'
```

Comment:
I think `style.css` is more common. @GRFreire could you rename the `.css` ?
```suggestion
css='<link rel="stylesheet" href="style.css">'
```